src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -24,6 +23,7 @@ public partial class LibraryView : UserControl
         public LibraryView()
         {
             InitializeComponent();
+            AddButton.Content = Dynamo.Wpf.Properties.Resources.AddButton;
```

Comment:
Bind it on xaml, not to set on code. 

Can we aggregate all fixes in one PR, if each fix is not big, instead of a lot of tiny PRs?